Sources/CoreCommands/Options.swift

```diff
@@ -451,13 +491,27 @@ public extension Sanitizer {
             return
         }
 
-        throw StringError("valid sanitizers: \(Sanitizer.formattedValues)")
```

Comment:
This seems like a useful error we're losing here

Reply:
its actually not needed as it is part of the help string that is presented in this case